projects/lf-documentation/src/app/getting-started/getting-started.component.html

```diff
@@ -1,10 +1,46 @@
 <p class="documentation-title">Getting Started</p>
-
-<p class="documentation-header">Using Framework-Agnostic Custom Elements</p>
+<p> UI components can be used via CDN script or NPM library. Below we list instructions on how to use the UI components.</p>
+<div class="toc-container">
+<p class="toc-container-header"> Contents</p>
+<ul>
+  <li>
+    <a href="#cdn-usage">Using Via CDN</a>
+  </li>
+  <li>
+    <a href="#npm-usage">Using Via NPM</a>
+  </li>
+  <li>
+    <a href="#set-language">Setting UI Components Language</a>
+  </li>
+</ul>
+</div>
+<h2 id="usage">General Usage Tips</h2>
```

Comment:
I think I expect this after the other sections so the yhave it set up. Also this should be added to the TOC